addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -83,6 +83,11 @@ const threadPatch = {
             this.isLoadingAttachments = false;
         }
     },
+    get notifyOnLeave() {
+        // Skip notification if display name is unknown (might depend on
+        // knowledge of members for groups).
+        return Boolean(this.displayName);
+    },
```

Comment:
Ok for now, though probably we should silently fetch and show notification, this would be better UX

Reply:
The issue is that group channels compute their display name client side based on the local knowledge of the channel members. We've known for a while that this logic should be changed. We should definitively think about it later on.

Reply:
Yeah, this even a performance issue in large populate: https://github.com/odoo/odoo/pull/178982

Eventually I think we should store compute name in python, so that there's less processing in JS. Also the name of group chat should have a limit, like it enumerate up to the 5 oldest persons in group chat. But that's more a topic for another day ;)